Svc/SeqDispatcher/SeqDispatcher.fpp

```diff
@@ -0,0 +1,48 @@
+module components {
+    @ Dispatches command sequences to available command sequencers
+    active component SeqDispatcher {
+
+        include "SeqDispatcherCommands.fppi"
+        include "SeqDispatcherTelemetry.fppi"
+        include "SeqDispatcherEvents.fppi"
+
+        async input port seqRunIn: Svc.CmdSeqIn
+
+        output port seqRunOut: [types.CMD_SEQUENCERS_COUNT] Svc.CmdSeqIn
```

Comment:
It might be worth adding sequence dispatcher custom constant (SeqDispatcherSequencerCount?) for tracking the number of command sequencers, since some projects may want to separately reserve some command sequencers for specific uses. I.e. command sequencer 0 might be used for fault protection sequence dispatch.

Reply:
Sorry yeah this types.CMD_SEQUENCERS_COUNT was a leftover from our internal fork. Will do

Reply:
Done